pulp_rpm/app/models/repository.py

```diff
@@ -502,8 +505,8 @@ class RpmPublication(Publication, AutoAddObjPermsMixin):
     TYPE = "rpm"
     checksum_type = models.TextField(choices=CHECKSUM_CHOICES)
     compression_type = models.TextField(null=True, choices=COMPRESSION_CHOICES)
-    metadata_checksum_type = models.TextField(choices=CHECKSUM_CHOICES)
-    package_checksum_type = models.TextField(choices=CHECKSUM_CHOICES)
+    metadata_checksum_type = models.TextField(null=True, choices=CHECKSUM_CHOICES)
+    package_checksum_type = models.TextField(null=True, choices=CHECKSUM_CHOICES)
```

Comment:
This was needed because without being nullable, or being overridden by the code (and since we're prepping to remove them, we don't want to touch these values in code), the default value here is empty string.  And Pydantic gets upset if Pulp throws out an empty string instead of either a null value or one of the defined choices.